modules/web-console/frontend/app/components/grid-showing-rows/controller.js

```diff
@@ -39,9 +42,49 @@ export default class {
     applyValues() {
         if (!this.gridApi.grid.rows.length) {
             this.count = 0;
+            this.visible = 0;
+            this.selected = 0;
+            return;
+        }
+
+        this.count = this.gridApi.grid.rows.length;
+        this.visible = _.sumBy(this.gridApi.grid.rows, (row) => Number(row.visible));
+        this.selected = this.gridApi.selection.getSelectedCount();
```

Comment:
This way selected status will update only when data changes and not when user changes selection.

Reply:
I fixed that.